codegen/templates/tchannel_client.tmpl

```diff
@@ -116,14 +136,16 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 		deps.Default.ContextMetrics,
 		deps.Default.ContextExtractor,
 		&zanzibar.TChannelClientOption{
-			ServiceName:          serviceName,
-			ClientID:             "{{$clientID}}",
-			MethodNames:          methodNames,
-			Timeout:              timeout,
-			TimeoutPerAttempt:    timeoutPerAttempt,
-			RoutingKey:           &routingKey,
-			AltSubchannelName:    scAltName,
+			ServiceName:		  serviceName,
+			ClientID:			 "{{$clientID}}",
```

Comment:
nit: these used to be lined up and are now all over the place

Reply:
thanks, it was a tab verus space causing alignment issues. fixed.